### PR TITLE
workflows: update legacy formula exceptions

### PR DESCRIPTION
**Pull request by FnControlOption**
- `openssl@3.0` should be `openssl@3`
- Update latest `python` version to `3.10` (Ditto for `python-tk`)

### CODE REVIEW

**Comment by SMillerDev** [comment]
Openssl 1 shouldn't be legacy until we move everything I think

**Comment by MikeMcQuaid** [comment]
> Openssl 1 shouldn't be legacy until we move everything I think

Agreed.

File: .github/workflows/triage.yml [hunk 1]
```diff
@@ -63,10 +63,9 @@ jobs:
                     "label": "legacy",
                     "path": "Formula/.+@.+",
                     "except": [
-                        "Formula/openssl@1.1.rb",
-                        "Formula/openssl@3.0.rb",
-                        "Formula/python@3.9.rb",
-                        "Formula/python-tk@3.9.rb"
+                        "Formula/openssl@3.rb",
```
Comment (MikeMcQuaid) [inline .github/workflows/triage.yml:66]:
```suggestion
                        "Formula/openssl@1.1.rb",
```

**Review by carlocab — APPROVED** [review]

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks @FnControlOption!

**Pull request merged**